spec/features/admin/tags_spec.rb

```diff
@@ -3,16 +3,21 @@
 feature 'Admin tags' do
 
   background do
-    @tag1 = create(:tag)
+    @tag1 = create(:category)
     login_as(create(:administrator).user)
   end
 
   scenario 'Index' do
     create(:debate, tag_list: 'supertag')
+    debate_tag = ActsAsTaggableOn::Tag.find_by(name: "supertag")
     visit admin_tags_path
 
     expect(page).to have_content @tag1.name
-    expect(page).to have_content 'supertag'
+    if debate_tag.category?
```

Comment:
@entantoencuanto I can't understand the usage of this if, could you please explain a bit? Thanks! :)